Index: local.properties
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>## This file must *NOT* be checked into Version Control Systems,\n# as it contains information specific to your local configuration.\n#\n# Location of the SDK. This is only used by Gradle.\n# For customization when using a Version Control System, please read the\n# header note.\n#Tue Feb 28 18:03:09 CET 2023\nsdk.dir=C\\:\\\\Users\\\\Caroline V\\\\AppData\\\\Local\\\\Android\\\\Sdk\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>ISO-8859-1
===================================================================
diff --git a/local.properties b/local.properties
--- a/local.properties	
+++ b/local.properties	
@@ -5,4 +5,4 @@
 # For customization when using a Version Control System, please read the
 # header note.
 #Tue Feb 28 18:03:09 CET 2023
-sdk.dir=C\:\\Users\\Caroline V\\AppData\\Local\\Android\\Sdk
+sdk.dir=C\:\\Users\\anouk\\AppData\\Local\\Android\\Sdk
